### ISSUE TITLE
cannot install wth pip or from source

### ISSUE THREAD

**Issue by beew** [axcore/tartube#107]
Tried to install tartube with pip encountered these error messages
```
ERROR: Could not find a version that satisfies the requirement gi (from tartube) (from versions: none)
ERROR: No matching distribution found for gi (from tartube)

```

Tried to install from source with sudo python3 setup.py install

```
Searching for gi
Reading https://pypi.org/simple/gi/
No local packages or working download links found for gi
error: Could not find suitable distribution for Requirement.parse('gi')
```
OS is ubuntu 20.04, python3-gi 3.36.0 is already installed.

**Comment by axcore** [comment]
Confirmed, installation via PyPI fails on both Ubuntu 20.04 and Linux Mint 19.3.

I don't know what the problem is. Exactly the same installation methods worked perfectly on both systems until recently. The code has not changed, so presumably the fault is somewhere else.

As a workaround, you can just download the source code and run it directly. In the directory containin setup.py, run

`python3 tartube/tartube`

**Comment by ayagrwl** [comment]
when installing from source, changing the dependency from "gi" to "pgi" (which is listed in the setup.py file) worked for me. I still had to manually copy the icons folder to the installed directory though.

**Comment by axcore** [comment]
Fixed in v2.1.051, thanks to @ayagrwl's suggestion.
